candle-metal-kernels/src/lib.rs

```diff
@@ -0,0 +1,1317 @@
+#![allow(clippy::too_many_arguments)]
```

Comment:
Would be better to only have this on the functions that require it rather than at the file level.

Reply:
Since it affects pretty much all the API I put it there for conciseness.

I'd rather make those functions with less arguments tbh, but I don't like introducing too many custom structs to manipulate this API.

I'll make the change for clippy on the appropriate functions.